test/verify/check-networking-bridge

```diff
@@ -54,7 +54,7 @@ class TestBridge(NetworkCase):
 
         # Check that the configuration file has the expected sane name
         # on systems that use "network-scripts".
-        if "ubuntu" not in m.image and "debian" not in m.image and "coreos" not in m.image:
+        if m.image not in ["debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-2004", "fedora-coreos", "fedora-33"]:
```

Comment:
We know that *no* Debian/Ubuntu image uses that, so going back to an explicit "ubuntu-XXYY" list feels like a regression. You can combine substring matching and an `in` list.

Reply:
fixed